src/Module.ts

```diff
@@ -298,6 +300,33 @@ export default class Module {
 			get dynamicImporters() {
 				return dynamicImporters.sort();
 			},
+			get exportedBindings() {
+				const reexportBindings = [...reexportDescriptions.entries()].reduce<
+					Record<string, string[]>
+				>((previous, currentValue) => {
+					const [name, { source }] = currentValue;
+					previous[source] ??= [];
+					previous[source].push(name);
+					return previous;
+				}, {});
```

Comment:
`reduce` has a tendency to be hard to read, and also may have sub-optimal performance as compared to a for-of loop. Why not just something like this

```suggestion
        const reexportBindings: Record<string, string[]> = {};
        for (const [name, { source }] of reexportDescriptions.entries()) {
          reexportBindings[source] ??= [];
          reexportBindings[source].push(name);
        }
```

Reply:
Uh, it’s just a personal habit, `reduce` is more convenient. But i will change `reduce` to `for-of` to optimize performance.

Reply:
It seems `for-of` is more concise than `reduce` here, hhh.